docs/source/workflows/performance_workflows.rst

```diff
@@ -0,0 +1,100 @@
+Performance workflows
+=====================
+
+A performance workflow runs cross-validation analysis on a predictor configuration
+
+Registration and validation
+---------------------------
+
+A workflow is registered with a project and validated before it is ready for use.
+Once registered, validation occurs automatically.
+Validation status can be one of the following states:
+
+-  **Created:** The module has been registered with a project and has been queued for validation.
+-  **Validating:** The workflow is currently validating. The status will be updated to one of the subsequent states upon completion.
+-  **Invalid:** Validation completed successfully but found errors with the workflow.
+-  **Ready:** Validation completed successfully and found no errors.
+-  **Error:** Validation did not complete.
+   An error was raised during the validation process that prevented an invalid or ready status to be determined.
+   Validation of a workflow and all constituent modules must complete with ready status before the workflow can be executed.
+
```

Comment:
In the states listed above we use the word module sometimes and other times the word workflows. Can we use "modules and workflows" when it applies to both.

Reply:
sure thing, fixed that